Package.swift

```diff
@@ -22,13 +22,16 @@ let package = Package(
     products: [
         .library(
             name: "DeepLearning",
-            targets: ["DeepLearning"]),
+            targets: ["DeepLearning"/*, "third_party"*/]),
```

Comment:
`third_party` should not be part of the `DeepLearning` target, since we are not shipping it as part of `DeepLearning`.

Reply:
Yup! Put it here temporarily as at the time I couldn’t create an Xcode project using swift package manager, and just wanted to verify the tests were passing through our CI since I was having trouble running them locally.